storage/client/bucket_metadata_test.cc

```diff
@@ -0,0 +1,91 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "storage/client/bucket_metadata.h"
+#include <gmock/gmock.h>
+
+/// @test Verify that we parse JSON objects into BucketMetadata objects.
+TEST(BucketMetadataTest, Parse) {
+  std::string text = R"""({
+ "kind": "storage#bucket",
```

Comment:
Would it make sense to indent these JSON literal lines 4 spaces for readability?

Reply:
Sure, done.